code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -55,6 +55,7 @@
 	desc = "This piece of junk looks like something that could have been used 700 years ago. It feels slightly moist."
 	sawn_desc = "An extremely sawn-off Mosin Nagant, popularly known as an \"obrez\". There was probably a reason it wasn't manufactured this short to begin with."
 	weapon_weight = WEAPON_HEAVY
+	w_class = WEIGHT_CLASS_HUGE
```

Comment:
This actually only prevents the weapons being worn in suit storage. The variable above it already was preventing the gun from being used with a full off-hand.

Reply:
yeah, I have second thoughts about it, especially that you cannot reload reasonably now